[2024-11-13T03:09:56.077+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_Model_Training_and_Evaluation.evaluate_and_compare_task manual__2024-11-13T03:09:52.546080+00:00 [queued]>
[2024-11-13T03:09:56.093+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_Model_Training_and_Evaluation.evaluate_and_compare_task manual__2024-11-13T03:09:52.546080+00:00 [queued]>
[2024-11-13T03:09:56.094+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-11-13T03:09:56.094+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-11-13T03:09:56.094+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-11-13T03:09:56.112+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): evaluate_and_compare_task> on 2024-11-13 03:09:52.546080+00:00
[2024-11-13T03:09:56.116+0000] {standard_task_runner.py:55} INFO - Started process 205 to run task
[2024-11-13T03:09:56.119+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'DAG_Model_Training_and_Evaluation', 'evaluate_and_compare_task', 'manual__2024-11-13T03:09:52.546080+00:00', '--job-id', '289', '--raw', '--subdir', 'DAGS_FOLDER/modeling_and_eval_dag.py', '--cfg-path', '/tmp/tmp9jazlehg']
[2024-11-13T03:09:56.120+0000] {standard_task_runner.py:83} INFO - Job 289: Subtask evaluate_and_compare_task
[2024-11-13T03:09:56.192+0000] {task_command.py:388} INFO - Running <TaskInstance: DAG_Model_Training_and_Evaluation.evaluate_and_compare_task manual__2024-11-13T03:09:52.546080+00:00 [running]> on host b77d411e2e76
[2024-11-13T03:09:56.252+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=House_Price_Prediction Team
AIRFLOW_CTX_DAG_ID=DAG_Model_Training_and_Evaluation
AIRFLOW_CTX_TASK_ID=evaluate_and_compare_task
AIRFLOW_CTX_EXECUTION_DATE=2024-11-13T03:09:52.546080+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-11-13T03:09:52.546080+00:00
[2024-11-13T03:09:56.264+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/modeling_and_eval_dag.py", line 76, in evaluate_and_compare_models_callable
    raise ValueError("Required data for model evaluation not found in XCom")
ValueError: Required data for model evaluation not found in XCom
[2024-11-13T03:09:56.269+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=DAG_Model_Training_and_Evaluation, task_id=evaluate_and_compare_task, execution_date=20241113T030952, start_date=20241113T030956, end_date=20241113T030956
[2024-11-13T03:09:56.286+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 289 for task evaluate_and_compare_task (Required data for model evaluation not found in XCom; 205)
[2024-11-13T03:09:56.298+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-11-13T03:09:56.321+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
